src/pci_caps.cpp

```diff
@@ -172,6 +176,13 @@ void get_link_stat_cur_speed(struct pci_dev *dev, char *buff) {
             link_cur_speed = "16 GT/s";
             break;
 #endif
+
+#ifdef PCI_EXP_LNKSTA_CLS_32_0GB
```

Comment:
Where we expect this macro to be defined ?

Reply:
Constants are taken from: https://github.com/torvalds/linux/blob/master/include/uapi/linux/pci_regs.h for PCIe